packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3673,7 +3673,12 @@ export interface ReactionButtonProps extends ControlBarButtonProps {
 
 // @beta
 export interface ReactionButtonStrings {
+    clapButtonTooltipContent?: string;
```

Comment:
nit: keep using reaction syntax? (not big opinion on this)

Switch clap -> applause 

suggestion: applauseReactionTooltipContent

Reply:
Updated as suggested for other apis as well.